src/lib/components/SpiderChart.svelte

```diff
@@ -121,6 +121,37 @@
 		}
 		return points;
 	});
+
+	let tooltip: HTMLDivElement;
+	let hidden: HTMLDivElement;
+
+	function mouseOver(description: string) {
+		tooltip = document.createElement('div');
+		tooltip.textContent = description;
+		tooltip.style.backgroundColor = 'var(--cream)';
```

Comment:
pulled in some styles from the other PoC I have. something I haven't accounted for is the awkward placement of Dynamic 3 on the side:
<img width="730" alt="Screenshot 2024-11-12 at 12 27 20 PM" src="https://github.com/user-attachments/assets/f2da9d50-2c0e-4995-ad93-7ad431d83d23">

but I'd like to hold off fixing this until we've made further changes to the overall layout/content of this page